Makefile

```diff
@@ -45,7 +45,7 @@ GO_LDFLAGS=-ldflags '-s -w -X $(PKG)/version.Version=$(VERSION) -X $(PKG)/versio
 SHIM_GO_LDFLAGS=-ldflags '-s -w -X $(PKG)/version.Version=$(VERSION) -X $(PKG)/version.Revision=$(REVISION) -X $(PKG)/version.Package=$(PKG) -extldflags "-static"'
 
 TESTFLAGS_RACE=
-GO_GCFLAGS=
+GO_GCFLAGS= -gcflags=-trimpath=${GOPATH}
```

Comment:
does that work if `$GOPATH` is a list?

Reply:
No, this needs to be fixed. There needs to be a trim per gopath entry.

Also, this should trim “src” from he start of each path.

Reply:
I tried to fix as suggested. But in case of multiple `-trimpath` only last flag passed has its effect, rest have no effect.

Also I found issue in golang repo: https://github.com/golang/go/issues/22382, that confirms the issue.

//cc @mlaventure @stevvooe